src/pages/Yeseul/Main/Components/Feed/Feed.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import User from '../User/User';
+import Comment from '../Comment/Comment';
+import IconButton from '../Button/IconButton';
+import './Feed.scss';
+
+class Feed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputComment: '',
+      commentId: 0,
+      comments: [],
+    };
+  }
+
+  componentDidMount() {
+    const { feedId } = this.props;
+    fetch('/data/Yeseul/commentData.json', {
+      method: 'GET',
+    })
+      .then(comments => comments.json())
+      .then(comments => {
+        this.setState({
+          commentId: comments.length + 1,
+          comments: comments.filter(comment => comment.feedId === feedId),
+        });
+      });
+  }
+
+  handleInput = e => {
+    this.setState({ inputComment: e.target.value });
+  };
+
+  addComment = e => {
+    const { inputComment, commentId, comments } = this.state;
+
+    e.preventDefault();
+    this.setState({
+      inputComment: '',
+      commentId: commentId + 1,
+      comments: [
+        ...comments,
+        {
+          id: commentId.toString(),
+          writer: this.props.userName,
+          content: inputComment,
+          tagId: '',
+        },
+      ],
+    });
+  };
+
+  deleteComment = clickedId => {
+    const { comments } = this.state;
+    this.setState({
+      ...this.state,
+      comments: comments.filter(comment => comment.id !== clickedId),
+    });
```

Comment:
- setState 가 기존의 state 값에 **병합** 하는 오퍼레이션 이기 때문에, ...this.state 를 해주실 필요가 없습니다.
```suggestion
    this.setState({
      comments: comments.filter(comment => comment.id !== clickedId),
    });
```